beacon_node/beacon_chain/tests/block_verification.rs

```diff
@@ -1339,7 +1354,7 @@ async fn verify_block_for_gossip_slashing_detection() {
         )
         .await
         .unwrap();
-    unwrap_err(harness.chain.verify_block_for_gossip(block2).await);
+    unwrap_err(harness.chain.verify_block_for_gossip(block2, 0).await);
```

Comment:
same as the earlier comment about CGC. It would never be 0 in Fulu.

Reply:
2ba26543e5f039b1ee9774bb1c4c2f21dfa2a5e1